src/components/Modal.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+
+import {
+  ModalStyle,
+  CloseButton,
+  DataFlex,
+  MoviePic,
+  ModalH1,
+  ModalH3,
+  ModalH5,
+  ModalSmall,
+  ModalDiv,
+  ModalDivSeat,
+  ModalSeatH1,
+  ModalScreen,
+} from "../styles/Booking.styled";
+
+import close from "../Asset/Xclose.png";
+import { useParams } from "react-router";
+const imgUrl = "https://image.tmdb.org/t/p/w500/";
```

Comment:
There is already a constant file to maintain all the urls, please move this variable to that file.